src/backend/mpi/swdsm.cpp

```diff
@@ -150,70 +151,62 @@ unsigned long isPowerOf2(unsigned long x){
   return retval;
 }
 
-void flushWriteBuffer(void){
-	unsigned long i,j;
-	double t1,t2;
-
-	t1 = MPI_Wtime();
+void flush_write_buffer(){
```

Comment:
Could this function be part of the WriteBuffer class?

Reply:
`flush_write_buffer` is now `WriteBuffer::flush()`.
`write_back_write_buffer` is now `WriteBuffer::flush_partial()`
`add_to_write_buffer` is now `WriteBuffer::add(T val)`
`WriteBuffer::erase(T val)` is used to erase a single element.